modules/appeals_api/app/controllers/appeals_api/v1/decision_reviews/notice_of_disagreements/evidence_submissions_controller.rb

```diff
@@ -14,7 +14,8 @@ class EvidenceSubmissionRequestValidatorError < StandardError; end
 
         def create
           status, error = AppealsApi::EvidenceSubmissionRequestValidator.new(params[:nod_uuid],
-                                                                             request.headers['X-VA-SSN']).call
+                                                                             request.headers['X-VA-SSN'],
+                                                                             'notice_of_disagreement').call
```

Comment:
Why pass this in as snake case only to camelize in the initializer? Can't we just pass in the proper string format since it's not being supplied from anything the consumer hands us (like a param)?

Reply:
fair point, this was just a first pass. I'll change it